Generate_Response/Check_POSTags.py

```diff
@@ -0,0 +1,33 @@
+""" Use POS tags to find useful sentence parts"""
```

Comment:
Correct me if I am wrong I think python module name should follows lowercase snake case convention (just like function names!).

So in this case, it should be
`generate_response/check_pos_tags.py`